test/integration/action/action-integration.spec.js

```diff
@@ -402,6 +402,13 @@ describe('Action Integration Tests', function() {
       expect(store1.computedChannels[0].status, 'to be', 'open');
     });
 
+    it('should have opened channels be private', async () => {
+      const { channels } = await grpc1.sendCommand('listChannels');
+      for (let channel of channels) {
+        expect(channel.private, 'to be', true);
+      }
+    });
+
```

Comment:
Great that you added a check in the integration tests. The integration tests should stick to using higher level actions that are basically invoked by the UI, so we can kind of replicate the user flow without spinning up a DOM. I'd suggest just adding `expect(store1.computedChannels[0].private, 'to be', true);` to the previous test.